<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/ico/pizza.ico</normaloff>images/ico/pizza.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #fff;</string>
  </property>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>461</width>
     <height>501</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: #fff;
border: 1px solid #2D9CDB;
box-sizing: border-box;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="backButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #2D9CDB;
border:none;
color: #fff;}
QPushButton:hover{
background: #005da7;
}
QPushButton:pressed{
background:  #2D9CDB;
}</string>
    </property>
    <property name="text">
     <string>Назад</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="idForm">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameForm">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="fioLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 12px;
line-height: 14px;
color: #000000;
border:none;</string>
    </property>
    <property name="text">
     <string>№ товара</string>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>151</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 12px;
line-height: 14px;
color: #000000;
border:none;</string>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>291</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #2D9CDB;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>289</width>
       <height>99</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="descriptionEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>291</width>
        <height>101</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;Выберите пункт из содержания&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLineEdit" name="priceProdEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="priceProdLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>71</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 12px;
line-height: 14px;
color: #000000;
border:none;</string>
    </property>
    <property name="text">
     <string>Цена</string>
    </property>
   </widget>
   <widget class="QLabel" name="toPodLabel_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>81</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 12px;
line-height: 14px;
color: #000000;
border:none;</string>
    </property>
    <property name="text">
     <string>Содержание</string>
    </property>
   </widget>
   <widget class="QLabel" name="desLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>81</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 12px;
line-height: 14px;
color: #000000;
border:none;</string>
    </property>
    <property name="text">
     <string>Описание</string>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>111</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameDesForm">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>190</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>Выберите пункт из содержания</string>
    </property>
    <property name="dragEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="nameToProdlabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>71</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 12px;
line-height: 14px;
color: #000000;
border:none;</string>
    </property>
    <property name="text">
     <string>Название</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="priceToProdEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>190</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="priceToProdLabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>61</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 12px;
line-height: 14px;
color: #000000;
border:none;</string>
    </property>
    <property name="text">
     <string>Стоймость</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="imgEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>331</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="imglabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>380</y>
      <width>151</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 12px;
line-height: 14px;
color: #000000;
border:none;</string>
    </property>
    <property name="text">
     <string>Название изображения</string>
    </property>
   </widget>
   <widget class="QLabel" name="formatlabel">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>380</y>
      <width>61</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 12px;
line-height: 14px;
color: #000000;
border:none;</string>
    </property>
    <property name="text">
     <string>Формат</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>421</width>
      <height>81</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid #2D9CDB;</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>419</width>
       <height>79</height>
      </rect>
     </property>
     <widget class="QTextEdit" name="desProdEdit">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>421</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Arial'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="decALabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>151</width>
      <height>14</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>14</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>14</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">font-family: Arial;
font-size: 12px;
line-height: 14px;
color: #000000;
border:none;</string>
    </property>
    <property name="text">
     <string>Общее описание товара</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>440</y>
      <width>200</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #2D9CDB;
border:none;
color: #fff;}
QPushButton:hover{
background: #005da7;
}
QPushButton:pressed{
background:  #2D9CDB;
}</string>
    </property>
    <property name="text">
     <string>Сохранить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="formatBox">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>400</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="addListButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>151</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #2D9CDB;
border:none;
color: #fff;}
QPushButton:hover{
background: #005da7;
}
QPushButton:pressed{
background:  #2D9CDB;
}</string>
    </property>
    <property name="text">
     <string>Добавить в содержание</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delToListButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>350</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #2D9CDB;
border:none;
color: #fff;}
QPushButton:hover{
background: #005da7;
}
QPushButton:pressed{
background:  #2D9CDB;
}</string>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveListButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>350</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background: #2D9CDB;
border:none;
color: #fff;}
QPushButton:hover{
background: #005da7;
}
QPushButton:pressed{
background:  #2D9CDB;
}</string>
    </property>
    <property name="text">
     <string>Сохранить содержание</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
